pennylane/numpy/tensor.py

```diff
@@ -113,6 +113,43 @@ def __array_wrap__(self, obj):
         out_arr = tensor(obj, requires_grad=self.requires_grad)
         return super().__array_wrap__(out_arr)
 
+    def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+        # pylint: disable=no-member,attribute-defined-outside-init
+        outputs = [
+            i.view(onp.ndarray) if hasattr(i, "unwrap") else i for i in kwargs.get("out", ())
+        ]
+
+        if outputs:
+            outputs = tuple(outputs)
+            kwargs["out"] = outputs
+        else:
+            outputs = (None,) * ufunc.nout

```

Comment:
Hmm, maybe a comment for why the outputs need to be recast this way?

Reply:
You are just extracting the numpy arrays so you can use the native function, right?

Reply:
Yep, this approach comes from the example shown in the NumPy documentation: https://numpy.org/devdocs/user/basics.subclassing.html#array-ufunc-for-ufuncs

Reply:
Done!